tests/Validators/MaxDateTest.php

```diff
@@ -92,4 +77,47 @@ public function testThatErrorMessageIsReplaced()
 
         $this->assertEquals($expected, $message);
     }
+
+    public function testThatInvalidParameterDateFails()
+    {
+        $this->expectException(\Exception::class);
+
+        $this->validator->validateMaxDate(
+            'date',
+            '2007-07-07',
+            [
+                'invalid',
+            ],
+            $this->laravelValidator
+        );
+    }
+
+    public function testThatInvalidValueDatePasses()
```

Comment:
Ok, so now it definitely looks like you're wanting bad dates to pass. Why is that?

Reply:
>It should pass validation if the value is invalid. I think it's the job of a separate `date_format` validator to test if the date is invalid, not `max_date` (otherwise you'd end up with two validation errors when only one of them is applicable)

Reply:
>Fair enough. This behaviour is tested from memory

TIL quoting. Thanks :p